ccmlib/common.py

```diff
@@ -1036,3 +1035,8 @@ def get_installed_scylla_package_hash(source_file: Path):
         if line.startswith("hash="):
             current_hash = line.replace("hash=", "").strip()
     return current_hash
+
+def get_yaml_instance():
+    yaml = YAML()
+    yaml.default_flow_style = False
```

Comment:
nit, it's not "getting" an instance. it creates one. so i'd suggest just use
```py
yaml = YAML()
```
because `default_flow_style` is `False` if `typ` is `"rt"` by default. and in that case, 
```py
self.default_flow_style = False
```

see https://github.com/commx/ruamel-yaml/blob/master/main.py#L66 , https://github.com/commx/ruamel-yaml/blob/27f71ab3c9bbf604de36b1ab6e170065f37cc2cb/main.py#L100 and https://github.com/commx/ruamel-yaml/blob/27f71ab3c9bbf604de36b1ab6e170065f37cc2cb/main.py#L152

Reply:
changed it.

and I'll give it a run